static/lang/sv.json

```diff
@@ -62,9 +62,9 @@
         "hint": "Slå på npcmystifierare, byt namn på spelfigur på den aktiva scenen baserat på deras egenskaper om Ctrl (konfigurerbar) klickas när du lägger till spelfiguren scenen.",
         "name": "Slå på npcmystifierare."
       },
-      "npcMystifierAddRandomNumber": {
-        "hint": "Aktiverar att lägga till ett slumpmässigt nummer när man mystifierar npcs. Observera att om spelfigur eller karaktär redan har ett nummer som suffix kommer detta inte att fungera bra.",
-        "name": "Lägg till slumpmässigt nummer i namnet"
+      "npcMystifierAddRandomProperty": {
```

Comment:
Can't read the other languages, but the auto translated stuff you added is, um, not right. :)
Remove the changes to the Swedish file (and if the french and portuguese are also auto translated, remove those as well.)